arbitrum-docs/for-devs/concepts/public-chains.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: "Arbitrum chains: A conceptual overview"
+sidebar_label: Arbitrum chains
+description: A high level description of the Arbitrum chains available
+reader-audience: developers who want to build on Ethereum/Arbitrum
+content-type: concept
+---
+
+Arbitrum chains are Layer 2 solutions built on top of the Ethereum blockchain, designed to increase scalability and reduce costs. In this conceptual overview, we’ll learn about the different Arbitrum chains and how they relate to each other. We’ll describe the available Arbitrum production and testnet chains, their differences, and the technology stacks that these chains use. We will finally list the current validators in each chain, as well as the members of the data availability committee (DAC) in Arbitrum Nova.
+
+## What Arbitrum production chains are available?
+
+### Arbitrum One
+
+**Arbitrum One** is the flagship Arbitrum mainnet chain, an optimistic rollup chain running on top of Ethereum mainnet. It offers a unique combination of benefits including trustless security, compatibility with Ethereum, scalability, and minimum cost. It uses the [Nitro](/inside-arbitrum-nitro/inside-arbitrum-nitro.mdx) technology stack, which is a major upgrade of the technology, improving over the "Classic" stack in several ways such as advanced calldata compression, separate contexts for common execution and fault proving, Ethereum L1 gas compatibility, and more.
+
+### Arbitrum Nova
+
+**Arbitrum Nova** is another production chain that uses the AnyTrust technology stack, a variant use case of the Nitro technology. In general, AnyTrust lowers costs by accepting a mild trust assumption. Instead of requiring all Arbitrum nodes to have access to the data of every L2 transaction in the Arbitrum chain's inbox, AnyTrust relies on an external Data Availability Committee to store data and provide it on demand, reducing the costs of batching and posting all L2 transaction data to L1.
```

Comment:
```suggestion
**Arbitrum Nova** is another production chain that uses the AnyTrust technology stack, a variant use case of the Nitro technology. In general, AnyTrust lowers the transaction costs by accepting a mild trust assumption. Instead of requiring all Arbitrum nodes to have access to the data of every L2 transaction in the Arbitrum chain's inbox, AnyTrust relies on an external data availability committee to store data and provide it on demand, reducing the costs of batching and posting all L2 transaction data to L1.
```